refactor(console + mapi): remove old theme code

jourdiw opened the pull request:
## Issue

https://gravitee.atlassian.net/browse/APIM-3061

## Description

Remove console theme code from the front and back.

There should be no regressions.

## Additional context

<!-- Add any other context about the PR here -->
<!-- It can be links to other PRs or docs or drawing -->
<!-- Or reproduction steps in case of bug fix -->

<!-- Environment placeholder -->

🏗️ Your changes can be tested here and will be available soon:
      Console: [https://pr.team-apim.gravitee.dev/5634/console](https://pr.team-apim.gravitee.dev/5634/console)
      Portal: [https://pr.team-apim.gravitee.dev/5634/portal](https://pr.team-apim.gravitee.dev/5634/portal)
      Management-api: [https://pr.team-apim.gravitee.dev/5634/api/management](https://pr.team-apim.gravitee.dev/5634/api/management)
      Gateway v4: [https://pr.team-apim.gravitee.dev/5634](https://pr.team-apim.gravitee.dev/5634)
      Gateway v3: [https://pr.gateway-v3.team-apim.gravitee.dev/5634](https://pr.gateway-v3.team-apim.gravitee.dev/5634)

<!-- Environment placeholder end -->
<!-- Storybook placeholder -->
---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-cuzttuohya.chromatic.com)
<!-- Storybook placeholder end -->

jourdiw commented:
@loriepisicchio @gaetanmaisse What about applying this to 4.0 + 4.1 ?

ThibaudAV reviewed (commented):
haha maybe it's not juste a refactor but a BC too 🙈

ThibaudAV commented on gravitee-apim-console-webui/src/management/configuration/theme/theme.component.ts:
```diff
@@ -1,24 +0,0 @@
-  bindings: {
-    views: '<',
-  },
-  controller: 'ThemeController',
```
This is the theme for the portal ? 🤔

jourdiw replied:
It is 🙃 I have FP and have saved the portal related assets and have renamed them. It still builds and runs no error 😅

jourdiw committed:
refactor(console)!: remove old theme code

BREAKING CHANGE: Loading of custom console theme is removed

jourdiw committed:
refactor(mapi)!: remove console-related theme in back

BREAKING CHANGE: Theme entities no longer available to Console Configuration entity

jourdiw committed:
refactor(console): specify remaining theme assets as portal theme

loriepisicchio commented:
I agree with @ThibaudAV the commits should not only be `refactor` but probably indicate a breaking change. 

Also to answer your question, I would not apply on earlier version IMHO (I don't think it's worth taking the risk)

jourdiw commented:
After discussion with @loriepisicchio, @ThibaudAV, and @gaetanmaisse, this will stay on master and not be merged onto 4.0 + 4.1.

ThibaudAV commented on gravitee-apim-console-webui/src/components/theme/theme-element.directive.ts:
```diff
@@ -1,39 +0,0 @@
-  controller: ThemeElementController,
-};
-
-export default ThemeElementDirective;
```
You should also remove all use of this directive : find : `gv-Theme-Element`

jourdiw replied:
Done.

ThibaudAV reviewed (approved):
The code looks good, but I haven't tested the application yet.
